testsuite/gnat2goto/tests/Incomplete_dec/incomplete_dec.adb

```diff
@@ -0,0 +1,6 @@
+package body Incomplete_Dec is
+   procedure P (X : in out Partial_Dec) is
+   begin
+      X.A := X.A + 1;
```

Comment:
Could we include some assertion as well? So that CBMC gets to the symex as well.

Reply:
What do you mean? Something like:
procedure P (X : in out Partial_Dec) is
   Dummy : Integer := 41;
   begin
      X.A := X.A + 1;
      pragma Assert (X.A = 42);
end P;